jax/experimental/jax2tf/jax2tf.py

```diff
@@ -882,11 +883,14 @@ def _get_shape_from_tensor_or_array(x):
   return tuple(x.shape)
 
 
-def _reduce_window(jax_f, reducer, init_val, operand, window_dimensions,
-                   window_strides, padding, base_dilation, window_dilation,
-                   input_shape=None):
+def _specialized_reduce_window(jax_f, reducer, identity, operand,
+                               window_dimensions, window_strides, padding,
+                               base_dilation, window_dilation,
+                               input_shape=None):
   """TensorFlow implementation of reduce_window_{sum,min,max}."""
```

Comment:
Please write a docstring documenting the usage

Reply:
Addressed in `3b8ae7ab`